pyproject.toml

```diff
@@ -10,7 +10,12 @@ license = { file = "LICENSE" }
 keywords = ["validate", "protobuf", "protocol buffer"]
 requires-python = ">=3.8"
 classifiers = [
-  "Programming Language :: Python :: 3",
+  "Programming Language :: Python :: 3.8",
+  "Programming Language :: Python :: 3.9",
+  "Programming Language :: Python :: 3.10",
+  "Programming Language :: Python :: 3.11",
+  "Programming Language :: Python :: 3.12",
+  "Programming Language :: Python :: 3.13",
```

Comment:
Not sure if we have a preference; I typically follow what projects like [structlog are doing](https://github.com/hynek/structlog/blob/f3e6fdee0658e9c6ab89aaa8823abe0d78e49b3a/pyproject.toml#L20-L25), which lists out the individual supported versions (dropping them when they're no longer supported). Happy to back this out, though, and leave it implicit from the `>=3.8`.